### DISCUSSION TITLE
Datamart Fase 1 - Correspondencia por tipos

### DISCUSSION THREAD

**Discussion by NestorFGG** [SINCOSOFT-SAS/SGD#35] [General]
DataMart que permitirá obtener información detallada de los registros de correspondencia entrante y facturas, teniendo en  cuenta los pasos de los flujos programados, descriptores y fechas

**Comment by NestorFGG** [comment]
Reunión 7 de diciembre de definición de campos fase 1

**Comment by aleja550** [comment]
Esta versión generada no era realmente un datamart, y su escalamiento no era posible debido a que no fue construido y concebido bien desde un inicio por lo cual, para escalar a la segunda fase, se volvió a hacer. Pero este producto no se entrego.
